packages/kokkos/core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -242,6 +242,11 @@ void *CudaUVMSpace::impl_allocate(
     const char *arg_label, const size_t arg_alloc_size,
     const size_t arg_logical_size,
     const Kokkos::Tools::SpaceHandle arg_handle) const {
+#ifndef KOKKOS_ENABLE_CUDA_UVM
```

Comment:
Do we want to push these changes into the develop branch?

Reply:
@kddevin No, we definitely don't want that to go into develop. I just rolled it back for this PR.